extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongodbConfig.java

```diff
@@ -71,11 +71,17 @@
     @ConfigItem(name = "native.dns.server-port", defaultValue = "53")
     public OptionalInt dnsServerPortInNativeMode;
 
-    // mongo.dns.lookup-timeout
     /**
      * If {@code native.dns.use-vertx-dns-resolver} is set to {@code true}, this property configures the DNS lookup timeout
      * duration.
      */
     @ConfigItem(name = "native.dns.lookup-timeout", defaultValue = "5s")
     public Duration dnsLookupTimeoutInNativeMode;
+
+    /**
+     * If {@code native.dns.use-vertx-dns-resolver} is set to {@code true}, this property enables the logging ot the
+     * DNS lookup. It can be useful to understand why the lookup fails.
+     */
+    @ConfigItem(name = "native.dns.log-activity", defaultValue = "false")
+    public Optional<Boolean> dnsLookupLogActivityInNativeMode;
```

Comment:
Is this really a native mode only option?

Reply:
Yes, we had to replace the dns lookup in native (via substitutions), because it was using jndi. I replaced it with some vertx code. However, dns lookup is sometimes close to rocket science, and to understand what's going on, we may need the dns frames.

Reply:
Thanks for the information!

So even if set in JVM mode, no logging information would be displayed?

Reply:
No, in JVM it uses ... JNDI. Nothing we can do, the Mongo client does not allow changing this default behavior.